examples/simple-console/src/main/java/com/snowplowanalytics/Main.java

```diff
@@ -45,9 +45,15 @@ public static void main(String[] args) {
         // build an emitter, this is used by the tracker to batch and schedule transmission of events
         BatchEmitter emitter = BatchEmitter.builder()
                 .url(collectorEndpoint)
-                .bufferSize(4) // send batches of 4 events. In production this number should be higher, depending on the size/event volume
+                .batchSize(4) // send batches of 4 events. In production this number should be higher, depending on the size/event volume
                 .build();
 
+        // For versions below 0.12.0, the batchSize property was called bufferSize
+//        BatchEmitter emitter = BatchEmitter.builder()
+//                .url(collectorEndpoint)
+//                .bufferSize(4) // send batches of 4 events. In production this number should be higher, depending on the size/event volume
+//                .build();
+
```

Comment:
I wouldn't add this comment. This is the example available with the current version so it's fine that it uses the current API. If we are concerned about people not understanding the change we can deprecate the old method in the BatchEmitter so it's still available. IMO a migration guide in the doc is more than enough.